README.md

```diff
@@ -748,9 +748,6 @@ widget.versions.last.changeset
 # {}
 ```
 
-The `object_changes` are only stored for creation and updates, not when an
```

Comment:
Hi @jaredbeck, does this look good to you?  I verified on my local machine.

Reply:
Yeah, deleting this sentence would be correct. Looks like this change was introduced by #1123 and released in 10.0.0.

Alternatively, this sentence could say: "Prior to 10.0.0, the `object_changes` were only stored for create and update events. As of 10.0.0, they are stored for all three events." Given that 10.0.0 is somewhat recent (1.5 yr), I don't mind having this much detail in the readme.

Reply:
@tlynam @jaredbeck 👍 I'll update to mention that, and link to the 10.0.0 changelog entry as well. One more question, the example shows that post-destroy, the changeset is an empty hash:

```
widget.destroy
widget.versions.last.changeset
# {}
```

but when I do it locally on my machine (running 10.3.0), I get `nil` instead. Should that be updated as well? See below:

```
2.6.4 :005 > s.versions.last
 => #<PaperTrail::Version id: 5401922, item_type: "Startup", item_id: 15766, event: "destroy", whodunnit: nil, object: "---\nid: 15766\nname: XXX\nwebsite: https://www.xx...", created_at: "2020-03-30 15:16:17"> 
2.6.4 :006 > s.versions.last.changeset
 => nil 
```

Reply:
That's interesting it returns `nil`.  Do you have the `object_changes` column?  When I run it without the `object_changes` column it returns `nil` and returns an empty hash for when I do have the column.  I don't see it in this example so I'm suspecting this is why it's behaving this way.  `=> #<PaperTrail::Version id: 5401922, item_type: "Startup", item_id: 15766, event: "destroy", whodunnit: nil, object: "---\nid: 15766\nname: XXX\nwebsite: https://www.xx...", created_at: "2020-03-30 15:16:17">`

Reply:
Hi @FanaHOVA, would you like to update the text?  Were you able to see if you had the `object_changes` column?  If that causes `changeset` to return `nil`?